sympy/parsing/mathematica.py

```diff
@@ -975,6 +976,7 @@ def converter(expr):
         "Times": Mul,
         "Plus": Add,
         "Power": Pow,
+        "Rational": lambda *a: Rational(*a),
```

Comment:
Why not just 

```suggestion
        "Rational": Rational,
```

Reply:
No reason except for clarity that it's a two-argument function that takes its arguments in the same order as Mathematica. Probably better without the lambda-function, so I'll make that change.